RUFAS/biophysical/animal/digestive_system/enteric_methane_calculator.py

```diff
@@ -18,34 +20,38 @@ def calculate_calf_methane(methane_model: str | None, body_weight: float) -> flo
 
         Parameters
         ----------
-        methane_model: str | None
+        methane_models: dict[str, bool]
             Methane model used for methane emission calculations, including Mutian, Mills, IPCC.
         body_weight: float
             Body weight of the current animal, kg.
 
         Returns
         -------
-        float
-            The amount of methane emission for calf (g/day).
+        dict[str, float]
+            The amount of methane emission for calf (g/day) corresponding to each method.
 
         References
         ----------
         (Pattanaik et al., 2003)
 
         """
-        methane_emission = 0.0
-        if methane_model:
-            methane_emission = (
-                0.013 * (body_weight**0.75) * GeneralConstants.KCAL_TO_MJ
-            ) / GeneralConstants.MJ_CH4_TO_G_CH4
+        methane_results = {}
+        for model, usage in methane_models.items():
+            if usage:
+                methane_emission = (
+                    0.013 * (body_weight**0.75) * GeneralConstants.KCAL_TO_MJ
+                ) / GeneralConstants.MJ_CH4_TO_G_CH4
+                methane_results[model] = methane_emission
+            else:
+                methane_results[model] = 0
```

Comment:
```suggestion
        for model, selected in methane_models.items():
            if selected:
                methane_emission = (
                    0.013 * (body_weight**0.75) * GeneralConstants.KCAL_TO_MJ
                ) / GeneralConstants.MJ_CH4_TO_G_CH4
                methane_results[model] = methane_emission
            else:
                methane_results[model] = 0
```
The word "usage" is a bit ambiguous